plugins/artnet/ArtNetNode.cpp

```diff
@@ -1334,11 +1334,13 @@ void ArtNetNodeImpl::RDMRequestCompletion(
       // hopefully update the remote controller
       port->on_discover->Run();
     } else {
-      OLA_WARN << "ArtNet RDM request failed with code " << reply->StatusCode();
+      OLA_WARN << "Art-Net RDM request failed with code "
+               << reply->StatusCode();
     }
   } else {
     // the universe address has changed we need to drop this request
-    OLA_WARN << "ArtNet Output port has changed mid request, dropping response";
+    OLA_WARN << "Art-Net Output port has changed mid request, "
+             <<"dropping response";
```

Comment:
Can we have a space after << please. Also we'd generally just move the one word that doesn't fit down (i.e. response in this case).